README.md

```diff
@@ -1,13 +1,13 @@
 # GolangBCN Meetup Group talks
+
 Submit your talk or propose a topic creating an Issue
 
 # Talk Standards
 
-* Talks are ~45 minutes including Q & A.
-* Two talks per meetup.
-* We have an international audience, so we prefer talks in English.
+* Talks should be ~30 minutes long, demo included, with a ~5 minutes Q & A session.
```

Comment:
Those ~5 mins of Q&A are included on the ~30?

Reply:
nope

Reply:
It's not clear to me :)